### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by Sachindu-Nethmin**
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [x] Updated the spec
- [ ] Checked native-image compatibility

### CODE REVIEW

**Review by aashikam — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -14,17 +14,91 @@ The `ballerinax/spotify` package provides APIs to connect and interact with Spot
...
+
+1. Open the [Spotify Developer Portal](https://developer.spotify.com/dashboard).
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-spotify/main/docs/setup/resources/dashboad.png alt="Dashboad" style="width: 70%;">
+
+
+2. Click on the "Create an app" tab and select an existing project or create a new one for which you want API Keys and Authentication Tokens.
+<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-spotify/main/docs/setup/resources/app_overview.png alt="Spotify Developer Portal" style="width: 70%;">
+
+
+### Step 2: Set up user authentication settings
```
Comment (NipunaRanasinghe) [inline README.md]: As a convention we use sentence case for all the headings in documentation. Please apply same for all the headers
```suggestion
### Step 1: Create a Spotify developer project

1. Open the [Spotify Developer Portal](https://developer.spotify.com/dashboard).
<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-spotify/main/docs/setup/resources/dashboad.png alt="Dashboad" style="width: 70%;">


2. Click on the "Create an app" tab and select an existing project or create a new one for which you want API Keys and Authentication Tokens.
<img src=https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-spotify/main/docs/setup/resources/app_overview.png alt="Spotify Developer Portal" style="width: 70%;">


### Step 2: Set up user authentication settings
```

File: README.md [hunk 2]
```diff
@@ -14,17 +14,91 @@ The `ballerinax/spotify` package provides APIs to connect and interact with Spot
...
+final spotify:Client spotify = check new({
+    auth: {
+        token
+    }
+});
+```
+
+### Step 3: Invoke the connector operation
+
+Now, utilize the available connector operations.
```
Comment (NipunaRanasinghe) [inline README.md:91]: We need to add a simple Ballerina code in here to demonstrate a most basic and simple use case. 

Refer to the below sample related to the twitter connector
Screenshot 2024-12-09 at 23 37 34

**Review by NipunaRanasinghe — CHANGES_REQUESTED** [review]

**Comment by Sachindu-Nethmin** [comment]
@NipunaRanasinghe  need review.

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**